package.json

```diff
@@ -1,26 +1,74 @@
+  ],
+  "homepage": "https://typegraphql.com",
+  "bugs": {
+    "url": "https://github.com/MichalLytek/type-graphql/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MichalLytek/type-graphql.git"
+  },
+  "funding": [
+    {
+      "type": "github",
+      "url": "https://github.com/sponsors/TypeGraphQL"
+    },
+    {
+      "type": "opencollective",
+      "url": "https://opencollective.com/typegraphql"
+    }
+  ],
+  "license": "MIT",
   "author": {
     "name": "Michał Lytek",
     "url": "https://github.com/MichalLytek"
   },
+  "main": "./build/index.js",
+  "types": "./build/index.d.ts",
+  "files": [
+    "./build"
+  ],
   "scripts": {
-    "test": "jest --verbose --coverage",
-    "test:ci": "jest --verbose --coverage --ci --forceExit --detectOpenHandles --runInBand",
-    "test:watch": "jest --watch",
-    "format": "prettier --write \"{src,tests,examples}/**/*.{ts,js}\" \"docs/**/*.md\"",
-    "check:format": "prettier --check \"{src,tests,examples}/**/*.{ts,js}\" \"docs/**/*.md\"",
-    "check:type": "tsc --noEmit && tsc --noEmit -p ./examples/tsconfig.json",
-    "check": "npm run check:format && npm run check:type",
-    "lint": "echo TODO: add ESLint",
-    "verify": "npm run check && npm run lint",
-    "package": "echo TODO: bash script",
-    "docs": "npm run --prefix website start"
-  },
-  "peerDependencies": {
-    "class-validator": ">=0.14.0",
-    "graphql": "^16.6.0"
+    "build": "npx tsc --build tsconfig.json",
+    "build:benchmarks": "npx tsc --build ./benchmarks/tsconfig.json",
+    "build:examples": "npx tsc --build ./examples/tsconfig.json",
+    "build:watch": "npx tsc --build --watch tsconfig.json",
+    "check": "npx npm-run-all --npm-path npm check:*",
+    "check:format": "npx prettier --check .",
+    "check:lint": "npx eslint .",
+    "check:markdown": "npx markdownlint \"**/*.md\"",
+    "check:script": "npx shellcheck ./.husky/pre-commit",
+    "check:spell": "npx cspell lint --config cspell.json --no-progress --show-context \"**\"",
+    "check:type": "npx tsc --noEmit && npx tsc --noEmit --project ./benchmarks/tsconfig.json && npx tsc --noEmit --project ./examples/tsconfig.json",
+    "clean": "npx shx rm -rf build",
+    "docs": "npm run --prefix website start",
+    "fix": "npx npm-run-all --npm-path npm fix:*",
+    "fix:format": "npx prettier --write .",
+    "fix:lint": "npx eslint --fix .",
+    "fix:markdown": "npx markdownlint --fix \"**/*.md\"",
+    "postbuild": "npx shx rm ./build/browser-shim.d.ts && npx shx cp ./src/browser-shim.ts ./build",
+    "prebuild": "npm run clean",
+    "prebuild:benchmarks": "npm run clean",
+    "prebuild:examples": "npm run clean",
+    "prebuild:watch": "npm run clean",
+    "prepare": "npx ts-patch install -s && npx husky install",
+    "prepublishOnly": "npm run build",
+    "test": "npx jest --verbose --coverage",
+    "test:ci": "npx jest --verbose --coverage --ci --forceExit --detectOpenHandles --runInBand",
```

Comment:
Why `detectOpenHandles` on CI? This is for debugging only as it slows down the execution.